base/timer/lap_timer_unittest.cc

```diff
@@ -49,6 +49,7 @@ TEST(LapTimer, UsageExample) {
   EXPECT_TRUE(timer.IsWarmedUp());
 }
 
+#if !defined(COBALT_PENDING_CLEAN_UP)
 #if !BUILDFLAG(IS_IOS) && !defined(COMPILER_MSVC)
```

Comment:
You can replace the `!defined(COMPILER_MSVC)` with `!defined(COBALT_PENDING_CLEAN_UP)` directly

Reply:
Will do. Curious what is this macro ``defined(COMPILER_MSVC)`` for?

Reply:
Previously it was only failing on windows so I disabled it because it seems we've implemented windows to just return 0 for threadticks in these calls. It was intentional, but I don't understand why it was done so it would be good to investigate more. See https://github.com/youtube/cobalt/blob/c2dabc423bea93755ef76f09ce0c2993293df2de/starboard/common/time.cc#L50